python/sdk/prelude_sdk/controllers/iam_controller.py

```diff
@@ -155,3 +155,17 @@ def audit_logs(self, days: int = 7, limit: int = 1000):
         if res.status_code == 200:
             return res.json()
         raise Exception(res.text)
+
+    @verify_credentials
+    def notifications(self, add: list = None, remove: list = None):
+        """ Modify your email notifications for specific events """
+        data = dict(add=add or [], remove=remove or [])
+        res = requests.post(
+            f'{self.account.hq}/iam/notifications',
```

Comment:
Instead of POST’ing a list of add/remove, I think I’d prefer ```POST:/iam/subscribe/12``` and ```DELETE:/iam/subscribe/12```, where 12 = the specific event key. This would line up with the intended interface as well, which will be toggle buttons, where we will fire off an API request on toggle (vs waiting for all toggles and clicking save).

Reply:
I can do that. I often bias toward the CLI where sending a list was best imo

I'll probably split into two lambdas again (AddNotificationFunction / DeleteNotificationFunction) unless you have a strong preference for keeping it as one lambda still

Reply:
Subscribe or Notify though :)

Reply:
(un/)subscribe it is. also sending through the enum name instead of int value since that's what we're moving toward with the other enums